experiment/src/org/labkey/experiment/api/FieldsTable.java

```diff
@@ -7,11 +7,11 @@
 import org.labkey.api.data.SQLFragment;
 import org.labkey.api.exp.query.ExpSchema;
 
-public class FieldNamesTable extends BaseFieldNamesTable
+public class FieldsTable extends BaseFieldsTable
 {
-    public FieldNamesTable(@NotNull ExpSchema userSchema, @Nullable ContainerFilter containerFilter)
+    public FieldsTable(@NotNull ExpSchema userSchema, @Nullable ContainerFilter containerFilter)
     {
-        super("FieldNames", userSchema, containerFilter);
+        super("Fields", userSchema, containerFilter);
```

Comment:
Haven't tested this, but I think it should compile and work as-is.

```suggestion
        super(ExpSchema.TableType.Fields.name(), userSchema, containerFilter);
```